### PR TITLE
added new testing scripts in package.json

### PR DESCRIPTION
**Pull request by rolo05**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,11 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix"
+    "lint:nofix": "vue-cli-service lint --no-fix",
+    "test:unit": "vue-cli-service test:unit",
+    "test":"npm run lint && npm run test:unit",
+    "test:unit:debug": "node --inspect-brk ./node_modules/jest/bin/jest.js --no-cache --runInBand"
```
Comment (Cyclodex) [inline package.json:12]: @rolo05 Please add the new task to the readme with short explanation for what it is

**Review by Cyclodex — CHANGES_REQUESTED** [review]
Please add the new task to the readme with short explanation for what it is

File: tests/unit/example.spec.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ describe("HelloWorld.vue", () => {
     const wrapper = shallowMount(HelloWorld, {
       propsData: { msg },
     });
+    debugger;
```
Comment (Cyclodex) [inline tests/unit/example.spec.ts]: I removed the debugger already

File: package.json (hunk 1 above)
Comment (rolo05) [reply package.json:12]: @Cyclodex  I have completed the readme and fixed naming on package.json

**Pull request merged**